op-tests/test-modpow.txt

```diff
@@ -1,100 +1,100 @@
 ; This file was generated by tools/src/bin/generate-modpow-tests.rs
 
-modpow 9341829497586649593865894363733490755438507694201597 556606291647424564814177367572458554425074604463581598568935778451121529 4373238041963667894698452839741832172073 => 645485963992042551624174571461937584879 | 26775
```

Comment:
how come this changed, if the generation is deterministic?

Reply:
right, you changed how the bytes are interpreted

Reply:
I also switched out the RNG